state-chain/pallets/cf-environment/src/lib.rs

```diff
@@ -36,7 +36,7 @@ pub mod weights;
 pub use weights::WeightInfo;
 pub mod migrations;
 
-pub const PALLET_VERSION: StorageVersion = StorageVersion::new(11);
+pub const PALLET_VERSION: StorageVersion = StorageVersion::new(12);
```

Comment:
where is the migration that moves this to v12? The placeholder migration is only at 11.

Reply:
ok, so the storage version should be at 11. I traced the change and I remember that this was bumped in https://github.com/chainflip-io/chainflip-backend/pull/5023 which was incorrect since the storage version didnt need to be bumped.

Reply:
perfect thanks for checking and resolving 👌